packages/edit-site/src/components/site-icon/style.scss

```diff
@@ -1,10 +1,19 @@
 .edit-site-site-icon__icon {
 	fill: currentColor;
+	// Make the WordPress icon not so small.
+	padding: $grid-unit-05 * 0.5;
+	// Matches SiteIcon motion, smoothing out the transition.
+	transition: padding	0.3s ease-out;
```

Comment:
When using transitions, animations must be opted out through the user's OS settings. The following description would be appropriate. We will need the following mixin.

```scss
@include reduce-motion("transition");
```

Instructions on how to disable animations can be found here: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences

Reply:
Added in https://github.com/WordPress/gutenberg/pull/58472/commits/aacb04a314dc09bd8662e7d34a8f8a62f79e97a3